### PR TITLE
Merge Prettified Corax+ Code

### PR DESCRIPTION
**Pull request by GamingMadster**
Here's the code cleanup I mentioned doing in #22.

- Added spacing for the code so the code was a bit easier to read through
- Removed unnecessary comments, added a couple others
- Made labels a bit more consistent throughout

I might try and do the same for some of the other source files, just so it's easier for someone looking to contribute to be able to read through the code.

### CODE REVIEW

**Comment by Timendus** [comment]
The Corax+ test is the worst offender of the bunch, I think. Primarily because I've initially tried to keep it just the "stock" Corax89 ROM with a few patches / bug fixes. I've added more and more changes over the years, without ever really taking the time to refactor.

I think your PR is an improvement overall, couple of remarks:

- Your version is removing a newline at the end of the file. I'm guessing your editor is configured to remove it. Just like the tabs/spaces discussion, this is a very subjective thing, and I generally just conform to whatever the rest of the project I'm contributing to uses.
- I like adding whitespace for readability. I'll go through later to see if I agree with your groupings.
- I don't see any changes to labels..?

All in all: congratulations on a much cleaner and much easier to review pull request than the last one! Great to see you learning in real time :smile:

**Comment by GamingMadster** [comment]
To be honest, combing through the code wasn't all too bad, but some areas were sure a doosie.

In terms of the labels, I think I'm combining the labels and comments for the start of each test in my mind, oops. 😅
I do think I still modified one though?

The whitespace i mainly did for the if statements that set i to image-no, although there are a couple other areas that I did it upon which you may deem unnecessary, unsure.

In terms of the newlines disappearing, I've just been using notepad to get the job done. I don't know why it would remove the newline at the end, since it's a simple text editor and it shouldn't auto format it. I guess I might have accidentally deleted it, oops x2. 😅

Oh, and thanks! Again, I'm glad to be contributing to the test suite!

File: src/tests/3-corax+.8o [hunk 1]
```diff
@@ -44,70 +44,95 @@
...
   drawop im1 imX
+
   i := image-ok
-  jump test1x
+  jump test-1X-pass
+
   i := image-no
-: test1x
+
+: test-1X-pass
```
Comment (GamingMadster) [inline src/tests/3-corax+.8o:127]: Okay, yeah, I knew I wasn't going crazy. This was the only label I changed. 😄

**Comment by GamingMadster** [comment]
Alright, I've done a little bit more to cleanup stuff.

- Did some small comment changes
- Added a few comments of my own
- Hopefully re-added the newline?

File: src/tests/3-corax+.8o [hunk 2]
```diff
@@ -32,13 +32,22 @@
...
 : test-2X-0E
   v0 := 1
   return
+
+  # These opcodes should only run if the interpreter doesn't have 00EE implemented.
+
+  # Note by Madster: I have seen one interpreter, of which I've forgotten the
+  # name of, that allows the execution of the opcode succeeding the return. There 
+  # is a chance that this would fail on that interpreter; however, I'll need to dig up that 
+  # interpreter to test that, though.
```
Comment (Timendus) [inline src/tests/3-corax+.8o:49]: Then that interpreter would be broken, and this test would successfully catch that issue ;)

**Comment by Timendus** [comment]
Thanks! I've removed many of the newlines again :P Sorry for that, but I do think it's a lot more readable now. Thanks for your help :)

**Pull request merged**